File: public/phase-2-20160620022203/C_PLUS_PLUS/flowers-girls-and-grandma.cpp
// Name : Flowers Girls and Grandma
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n :  the number of girls
	// p : Grandmas favourite prime number
	// lr : each row corresponds to the range of flowers that each girl can grow
	int idx, jdx;
	int n, p;

	cin >> n >> p;
	int lr[10000][2];

	for(idx=0;idx<n;idx++)
		for(jdx=0;jdx<2;jdx++)
			cin >> lr[idx][jdx];


	// write your code here
	// store your results in `result`

	// output
	// Dummy Data
	float result=0.0;

	cout << result << endl;

}

File: public/phase-2-20160620022203/C_SHARP/flowers-girls-and-grandma.cs
// Name : Flowers Girls and Grandma
using System;
namespace Application {
	class flowers_girlsand_grandma {
		public static void Main(string[] args) {
		// input:
		// n :  the number of girls
		// p : Grandmas favourite prime number
		// lr : each row corresponds to the range of flowers that each girl can grow
		int idx;
		int jdx;
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int p = Convert.ToInt32(elements[1]);
		int[,] lr = new int[n,2];
		string[] lr_elements = new string[100];
		for(idx=0;idx<n;idx++){
			lr_elements = (Console.ReadLine()).Split(' ');
			for(jdx=0;jdx<2;jdx++){
				lr[idx,jdx] = Convert.ToInt32(lr_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		float result=0.0;

		Console.WriteLine(result);

		}	
	}
}